specifyweb/context/views.py

```diff
@@ -602,3 +610,19 @@ def api_endpoints(request):
 def api_endpoints_all(request):
     """Returns a JSON description of all endpoints."""
     return JsonResponse(generate_openapi_for_endpoints(True))
+
+@require_http_methods(['GET', 'POST'])
+def language(request):
+    """Get List of available languages OR set current language."""
+    if request.method == 'GET':
+        return JsonResponse({
+            'data': {
```

Comment:
Why are you wrapping all of your JSON responses in `{"data": ...}`?

Reply:
There is no reason for now, but I have seen some APIs respond like `{error: false, data: ...}` or `{error: 'Out of range': data: ...}`, which seems nicer (and more consistent), rather than delivering error messages in plain text.
Also, I think json:api standard does something similar.

Reply:
IDK. For one thing, the error message is still plain text! So I don't see how it helps. And the type of the `error` parameter is `Union[str, Literal[False]]` which is funky. We have HTTP status codes for indicating that the request failed.

I think you are anticipating an `data Either = Ok Data | Failure Error` monad encoded in JSON, which is great. (See: https://en.wikipedia.org/wiki/Result_type) But, I wouldn't start including half implementations. If there is no reason for something now, then there is no reason for it. Since now is now, after all.

Reply:
Fixed in d1e305f7